src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -30,6 +30,8 @@ public class GenerateWasmBootJson : Task
     [Required]
     public bool DebugBuild { get; set; }
```

Comment:
Is this needed if we have `DebugLevel` being set? the caller can set this to 0/1 if it doesn't have anything else, and DebugBuild=true .

Reply:
The task is shared with .NET 6+7 where DebugLevel is not used by Blazor.